pkgs/development/tools/build-managers/gnumake/default.nix

```diff
@@ -1,4 +1,6 @@
-{ lib, stdenv, fetchurl, guileSupport ? false, pkg-config, guile }:
+{ lib, stdenv, fetchurl, guileSupport ? false, inBootstrap ? false, pkg-config, guile }:
+
+let guileEnabled = guileSupport && !inBootstrap; in
```

Comment:
use a assert?

Reply:
Can you elaborate? AFAIU `assert` would fail the evaluation. The intent here is not to fail `gnumake.override { guileSupport = true; }` evaluation (be it evaluated in the middle of bootstrap or as a leaf package).